apps/src/templates/SchoolNotFound.jsx

```diff
@@ -22,24 +22,8 @@ export default class SchoolNotFound extends Component {
     showErrorMsg: PropTypes.bool
   };
 
-  sendNameToParent = (event) => {
-    this.props.setField("schoolName", event.target.value);
-  }
-
-  sendTypeToParent = (event) => {
-    this.props.setField("schoolType", event.target.value);
-  }
-
-  sendStateToParent = (event) => {
-    this.props.setField("schoolState", event.target.value);
-  }
-
-  sendZipToParent = (event) => {
-    this.props.setField("schoolZip", event.target.value);
-  }
-
-  sendCityToParent = (event) => {
-    this.props.setField("schoolCity", event.target.value);
+  handleChange = (field, event) => {
+    this.props.onChange(field, event);
```

Comment:
I'd like it a _little_ better if we only passed the new value to the `onChange` callback like you did before, rather than passing the whole event object.  Up to you though, I know that adds a little complexity back to the other component.